### PR TITLE
[BREAKING] chore: upgrade to polkadot-v0.9.26

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

## Changes

https://github.com/paritytech/substrate/compare/polkadot-v0.9.24...polkadot-v0.9.26

- https://github.com/paritytech/substrate/pull/11490
- https://github.com/paritytech/cumulus/pull/1325
- https://github.com/paritytech/polkadot/pull/5671
- https://github.com/paritytech/substrate/pull/11618

## Dependencies

- https://github.com/open-web3-stack/open-runtime-module-library/pull/793
- https://github.com/shaunxw/xcm-simulator/pull/36

## Notes

- `SpendOrigin` set to `NeverEnsureOrigin` to disallow additional origins to approve treasury proposals
- Added `RelayNumberStrictlyIncreases` to `ParachainSystem`

### CODE REVIEW

File: parachain/src/cli.rs [hunk 1]
```diff
@@ -44,43 +44,11 @@ pub enum Subcommand {
...
-    #[clap(long)]
-    pub chain: Option<String>,
-}
-
 /// Command for exporting the metadata.
 #[derive(Debug, Parser)]
 pub struct ExportMetadataCommand {
     /// Output file name or stdout if unspecified.
-    #[clap(parse(from_os_str))]
+    #[clap(action)]
```
Comment (sander2) [inline parachain/src/cli.rs:51]: Why?

**Review by sander2 — CHANGES_REQUESTED** [review]
Should be marked breaking because the treasury pallet added a new extrinsic.

File: parachain/src/cli.rs (hunk 1 above)
Comment (gregdhill) [reply parachain/src/cli.rs:51]: Change in `clap-rs` [`v3.2.0`](https://github.com/clap-rs/clap/compare/v3.1.18...v3.2.0) [here](https://github.com/clap-rs/clap/pull/3794). See also the change in `cumulus` [here](https://github.com/paritytech/cumulus/commit/2e52ce29e91bcf34b3003810846c576bd4396575).

Comment (sander2) [reply parachain/src/cli.rs:51]: That didn't really tell me what exactly it did, so I did some googling and found this explanation for what `action` does: https://github.com/clap-rs/clap/discussions/4041

File: parachain/runtime/kintsugi/src/lib.rs [hunk 2]
```diff
@@ -639,6 +640,7 @@ impl pallet_treasury::Config for Runtime {
     type Currency = NativeCurrency;
     type ApproveOrigin = EnsureRoot<AccountId>;
     type RejectOrigin = EnsureRoot<AccountId>;
+    type SpendOrigin = frame_support::traits::NeverEnsureOrigin<Balance>;
```
[the remark this answers is not in the document]
Comment (gregdhill) [reply parachain/runtime/kintsugi/src/lib.rs:526]: To be honest, I don't think it really matters - but this is the change Acala also made

[the remark this answers is not in the document]
Comment (sander2) [reply parachain/runtime/kintsugi/src/lib.rs:526]: > I don't think it really matters

Can you explain why not? Why would we opt out of this feature? Wouldn't it turn the current 2-stage process into a more streamlined single stage process? I.e. no more need to first create a treasury proposal and then a governance proposal to enact the treasury proposal

**Review by sander2 — APPROVED** [review]

**Pull request merged**